src/gc.h

```diff
@@ -198,7 +175,7 @@ typedef struct {
 // We'll use this size to determine the size of the data stack corresponding to a
 // PC stack size. Since the data objects are not all of the same size, we'll waste
 // some memory on the data stack this way but that size is unlikely going to be significant.
-typedef union {
+union gc_mark_data {
```

Comment:
I think our usual convention is to call this `_gc_mark_data_t` and to define it on the same line as the typedef